distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -428,23 +336,12 @@ meta value = if is_atom value then Atom.Value value else
    - value_1: The first value.
    - value_2: The second value.
 is_same_object : Any -> Any -> Boolean
-is_same_object value_1 value_2 = @Builtin_Method "Meta.is_same_object"
-
-## PRIVATE
-   ADVANCED
-
-   Checks if `value` is an instance of `typ`.
-
-   Arguments:
-   - value: The value to check for being an instance of `typ`.
-   - typ: The type to check `self` against.
-is_a : Any -> Any -> Boolean
-is_a value typ = @Builtin_Method "Meta.is_a"
+is_same_object value_1 value_2 = @Tail_Call Meta_Helpers.is_same_object_builtin value_1 value_2
 
```

Comment:
Some methods are commonly used over the code base. Keep them?
- Even they usually have some alternative
- `Meta.is_same_object o1 o2`
- `o1.is_same_object_as o2` 
- should be identicalb

Reply:
I liked the `Meta.is_same_object` as it was slightly more readable (very subjective).

But I don't think there's any harm in rewriting to only use `o1.is_same_object_as o2`

Reply:
Let's keep both methods.